config/software/python-nghttp2.rb

```diff
@@ -0,0 +1,10 @@
+name "python-nghttp2"
+default_version "2.10.3"
+
+dependency "python"
+dependency "pip"
+
+build do
+  ship_license "https://raw.githubusercontent.com/andymccurdy/redis-py/master/LICENSE"
+  command "#{install_dir}/embedded/bin/pip install -I --install-option=\"--install-scripts=#{install_dir}/bin\" redis==#{version}"
```

Comment:
instead of `redis==#{version}` it should be `#{name}==#{version}`.

But it looks like there's no `pip` package for nghttp2. Isn't building nghttp2 with python bindings enough to use it from python?